src/components/MoviesList/MoviesList.js

```diff
@@ -4,10 +4,11 @@ import PropTypes from 'prop-types';
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
+export const MoviesList = ({ movies, filter }) => (
   <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} {...movie} />
+    {movies.filter(movie => movie.title.toLowerCase().includes(filter.toLowerCase()) // eslint-disable-line
+      || movie.description.toLowerCase().includes(filter.toLowerCase())).map(movie => ( // eslint-disable-line
+      <MovieCard key={movie.imdbId} {...movie} /> // eslint-disable-line
```

Comment:
почему бы не попереносить строки, примерно так, чтобы не использовать disable-line

```suggestion
    {movies.filter(movie => movie.title
      .toLowerCase()
      .includes(filter.toLowerCase()) 
        || movie.description
          .toLowerCase()
          .includes(filter.toLowerCase()))
            .map(movie => (
      <MovieCard key={movie.imdbId} {...movie} />
```

Reply:
good advice